<?xml version="1.0"?>
<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
  <joint name="camera_joint" type="fixed">
    <origin xyz="0.03087 0 0.09048" rpy="0 ${radians(20)} 0"/>
    <parent link="base_link"/>
    <child link="camera_link"/>
  </joint>

  <link name="camera_link">
    <collision>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <box size="0.002 0.02 0.02"/>
      </geometry>
    </collision>
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <box size="0.002 0.02 0.02"/>
      </geometry>
      <material name="laser_mat">
        <color rgba="0.0 0.0 0.0 0.8"/>
      </material>
    </visual>
  </link>
</robot>
